.github/workflows/rust.yml

```diff
@@ -241,6 +241,7 @@ jobs:
       - name: Verify that benchmark queries return expected results
         run: |
           export TPCH_DATA=`realpath datafusion/sqllogictest/test_files/tpch/data`
+          # use release build for plan verificaton because debug build causes stack overflow
```

Comment:
Hmm, do you know what the cause of stack overflow there under debug build?

Reply:
From https://github.com/apache/arrow-datafusion/issues/7709#issuecomment-1742050117 it appears that `round_trip_logical_plan_q21` fails for some reason. 

I did not debug it further.